frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,44 @@
+.errorMessage {
+  padding: 0;
+  margin: 0;
+  margin-bottom: var(--font-size-xxs);
+  font-size: var(--font-size-xxs);
+  line-height: var(--font-size-xs);
+  color: var(--color-red);
+}
+
+.input {
+  padding: var(--font-size-sm) var(--font-size-md);
+  font-family: var(--font-family-body);
+  font-size: var(--font-size-xs);
+  font-style: normal;
+  font-weight: 400;
+  line-height: var(--font-size-sm);
+  color: var(--color-black);
+  border: 1px solid var(--color-purple);
+  border-radius: 10px;
+  box-shadow: 0 4px 5px rgba(159, 24, 217, 50%);
+}
+
+.input:disabled {
+  color: var(--color-dark-gray);
+  background-color: var(--color-light-gray);
+  border: 1px solid var(--color-dark-gray);
+  box-shadow: 0 4px 5px var(--color-gray);
+}
+
+::placeholder {
+  color: var(--color-light-gray);
+}
```

Comment:
Thought I left a comment regarding this. I believe this will leak into other components if we don't use `.input`